src/main/java/org/variantsync/diffdetective/variation/diff/graph/GraphView.java

```diff
@@ -0,0 +1,81 @@
+package org.variantsync.diffdetective.variation.diff.graph;
+
+import org.variantsync.diffdetective.util.StringUtils;
+import org.variantsync.diffdetective.variation.diff.DiffNode;
+import org.variantsync.diffdetective.variation.diff.DiffTree;
+import org.variantsync.diffdetective.variation.diff.Time;
+
+import java.util.*;
+
+/**
+ * A view on a DiffTree that shows the DiffTree as a list of nodes and edges.
+ * The view invalidates as soon as the viewed DiffTree is altered as the view
+ * will not update itself.
+ *
+ * @param nodes The set of all nodes in a DiffTree.
+ * @param edges The set of all edges in a DiffTree.
+ *
+ * @author Paul Bittner
+ */
+public record GraphView(
```

Comment:
Yes but DiffTree's aren't views, right? But I agree that a `GraphView` is not necessarily a view. I will rename it to FormalDiffGraph, since the types are still fixed to `DiffNode`s.